README.md

```diff
@@ -1 +1,47 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a project that I learned about linter, HTML and css
+
+## Built With
+
+- HTML
+- Css
+
+
+## Getting Started
+
+**This is the first project using HTML and css and I learned about linter**
+
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
You should add the instructions to set up this project in any local environment, for instance: 

```
To get a local copy up and running follow these simple example steps.
- Open terminal
- Clone this project by the command `git clone URL/repository_name.git`
- `cd repository_name` folder
- Open `index.html` in your local browser or using Live Server in Visual Studio Code.
```

- [ ] Add more instructions about how to set up this project.